### ISSUE TITLE
Allow users to mark audio as reviewed

### ISSUE THREAD

**Issue by AndiPersti** [Tatoeba/tatoeba2#2517]
This was suggested by [mramosch](https://tatoeba.org/eng/user/profile/mramosch) in a PM.

Similar to reviewing sentences, it should be possible to review audio recordings. This review should be displayed on the sentence page.
In addition it may be useful to add a search filter in order to only find sentences with reviewed audio.

**Comment by mramosch** [comment]
> A native speaker can determine better if an audio recording is poorly pronounced, heavily accented or doesn't match the sentence, before it's imported. I remember recordings of some users being criticized for not sounding like a native speaker. I think it would be better to check them before importing than reviewing later.

Proof listening Audio is not only about pronunciation and intonation but also about technical issues like e.g. chopped off tails and heads. So I could easily check the technical aspects of sentences in other languages than my mother tongue.

However, after having listened to almost 6000 sentences of other contributors and having done a second pass on my own almost 7000 sentences there are still almost 20.000 left for me to check in German. So this is of course the preferred way to go when you listen for interpretation, intonation etc. and technical quality at the same time ;-)

However, if  there’s no way to indicate to other proof-listeners that bulks of recordings have already been checked, someone else might do the same work again, just to find out, that there are no flaws at all (optimal case).

There is also no dedicated GERMAN language homepage (and every other language for that matter) where German ‚staff‘ could announce these issues in a centralized manner (without having to hope for the right search results on the wall and having to read through tons of threads). Even not being language biased, Tatoeba offering such a facility would attract more active people to check regularly for news on their language and field of work.

When you‘re doing things big scale, every click counts, so there is no time for hitting 2-4 buttons just to confirm a checked audio recording. After finishing an audio list (like I did with gretelen‘s audio list of 5000+ sentences, because I found a lot of mistakes when doing a quick cross check on some of them) I wanna hit one single button that batch-marks all Audio recordings in one felt swoop, of course after all 298 chopped off recordings and 100 more that had to be moved to new sentences (created manually by me as alternative) are properly handled according to their problems.

So this is a different UI (functionality/workflow) than for the average Joe who wants to mark a recording as being Ok every once in a blue moon. This information is only really useful if it is provided big scale (for search criteria etc.) and I guess ‚read only‘ is enough for the average user. But for us big scale worker ‚write‘ needs to be much simpler because we are talking about tens of thousands of clicks more for a proof-listen session of 5000 sentences.

Moving a recording does not only mean renaming the file, but also correcting the text in the title field in the MP3 tag etc. and for hundreds of files this is not a feasible workflow. But also getting thousands of new recordings to proof-listen before they are imported, just as audio files without the context of the sentence page, is useless because sometimes you just need to read the source sentence for a translation and even listen to the source recording to make an informed decision about a recorded sentence, and that is only possible in an environment like the sentence page itself, and not with loose audio files.

My way around this missing functionality right now is to listen to all 30.000 German sentences and hoping for someone at Tatoeba to provide me with a list of all new German additions (in the same format than the User/audio list format) - so I can simply say, if being asked, ALL German recordings are proof-listened!

Where the rest of the community can find this information (without personally asking me or someone who happens to know that this work was already done by someone else) is still an unsolved issue...

———

**Comment by jiru** [comment]
@mramosch Thanks a million for thoroughly clarifying your actual need. This is very valuable information that you are giving us. About having a dedicated space for a community of a certain language to share and organize itself in its own language, this is something I have been thinking about for a while too. I can’t find anything on the Wall and on Github so I’ll just create a new issue.

**Comment by mramosch** [comment]
Would it be possible to have an option for e-mail notifications that owners of audio files could activate in order to be notified of new comments on their recorded sentences?

At the moment it is only the sentence owner and those who have already added a comment to the corresponding sentence. And of course explicitly those targeted with @USER.

Since people who find faulty audio files usually write a comment but do not tag (@audio: ISSUE) etc., the creator of the soundtrack cannot react or provide a correction or a replacement.

I think the audio environment should (even if it is still treated as a third-class member) receive a few small extensions to raise the awareness of the community a little bit that something is happening in Audioland ;-)

**Comment by jiru** [comment]
@mramosch Yes, of course. Please create new issues on Github for each of the improvements you’d like to see.
